backend/src/akvo/flow_services/geoshape_export.clj

```diff
@@ -19,56 +19,91 @@
             [akvo.commons.config :as config]
             [cheshire.core :as json]
             [taoensso.timbre :as timbre]
-            [akvo.flow-services.util :as util])
-  (:import [java.util UUID]))
+            [akvo.flow-services.util :as util]
+            [clojure.string :as s])
+  (:import [java.util UUID TimeZone]
+           (org.waterforpeople.mapping.dataexport.service BulkDataServiceClient)
```

Comment:
We usually (most of the time) use vectors instead of lists. It's usually more appealing since `(something x)` suggests like a function call.

Reply:
Fixed